mindsdb/interfaces/skills/custom/text2sql/mindsdb_kb_tools.py

```diff
@@ -223,6 +230,7 @@ def _run(self, tool_input: str) -> str:
         try:
             # Execute the query
             query = llm_str_strip(query)
+            query = self._remove_backticks_from_identifiers(query)
```

Comment:
this should brake there queries that have table names and column names with special characters 
It was fixed in this PR https://github.com/mindsdb/mindsdb/pull/11101
@MinuraPunchihewa, is it correct?

Reply:
Hmm.. @ea-rus Yes I believe that since querying KBs also call the `run_no_throw()` function, the above PR handles this already.